hello-oboe/app/src/main/cpp/hello-oboe.cpp

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2019 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <jni.h>
+#include "OboeSinePlayer.h"
+
+
+static OboeSinePlayer *oboePlayer = nullptr;
+
+
+extern "C" {
+    JNIEXPORT void JNICALL
```

Comment:
create() should return an error code if it fails. Then the Java UI can put up a Toast. But this can wait for a later CL.